docs/Deleting.md

```diff
@@ -1,3 +1,5 @@
+# Editions: deleting an issue
+
```

Comment:
I felt like this file could do with a heading, just to clarify what it was about